guides/doc-Managing_Hosts/topics/ref_customizing-the-global-registration-and-host-registration-templates.adoc

```diff
@@ -0,0 +1,94 @@
+[id="customizing-the-global-registration-and-host-registration-templates_{context}"]
+= Customizing the Global Registration and Host Registration Templates
+
+Use information in this section if you want to customize the registration process.
+
+Note that all default templates in {Project} are locked.
+If you want to customize the registration process, you need to clone the default Global registration template and change the *Default Global registration template* setting in *Administer* > *Settings* > *Provisioning*.
+
```

Comment:
Maybe I overlooked it, but for customizing the Host registration template you need to clone it as well and then you have to associate it with the OS of interest

Reply:
This is true, I added instructions on this Host registration template too.